addons/mail/static/tests/owl/form_renderer_chatter_tests.js

```diff
@@ -63,24 +89,180 @@ QUnit.test('basic chatter rendering', async function (assert) {
     );
 });
 
+QUnit.test('basic chatter rendering without followers', async function (assert) {
+    assert.expect(7);
+    this.data['res.partner'].records = [{
+        activity_ids: [],
+        id: 2,
+        display_name: "second partner",
+        message_ids: [],
+        message_follower_ids: [],
+    }];
+    await this.createView({
+        data: this.data,
+        hasView: true,
+        // View params
+        View: FormView,
+        model: 'res.partner',
+        arch: `
+            <form string="Partners">
+                <sheet>
+                    <field name="name"/>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="activity_ids" widget="mail_activity"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                </div>
+            </form>`,
+        res_id: 2,
+    });
+
+    assert.containsOnce(document.body, `.o_Chatter`,
+        "there should still be a chatter"
```

Comment:
Remove all the "still", it doesn't make sense when there wasn't an assertion without "still" before.